docs/source/examples/combine_ddd_zne.md

```diff
@@ -0,0 +1,192 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Composing techniques: Digital Dynamical Decoupling and Zero Noise Extrapolation
+
+Noise in quantum computers can arise from a variety of sources, and sometimes applying multiple error mitigation techniques can be more beneficial than applying a single technique alone. 
```

Comment:
It would be nice to give some intuition of when composing is beneficial and why in those cases it works. I see you touch on that later down in the notebook, but I think it's good to say it upfront. Ideally also citing a reference.

Reply:
I am not aware of any references which specifically combine DDD and ZNE error mitigation. I think a more extensive study would be needed to determine the exact circumstances under which particular QEM techniques are useful to compose. 

Section IV C. of this [QEM survey paper](https://arxiv.org/pdf/2210.00921#section*.30) details some combinations which have been used in the existing literature, but does not mention DDD in combination with any other techniques, nor does it compare the relative efficacy of the various methods.

The paper overall leaves it as an open question to determine the optimal combination of techniques for a given use case in [section VI](https://arxiv.org/pdf/2210.00921#section*.36). 

I can certainly add this paper as a reference; it just won't necessarily be useful for the specific topic of this tutorial.

Reply:
Thanks for those references! I agree that it will not be useful for the specific topic of DDD+ZNE. Time to write an article of our own on the topic :)

Reply:
I was thinking the same thing!